.github/workflows/ci.yaml

```diff
@@ -11,6 +11,10 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Check update docs
```

Comment:
I think it can be more simple, for example: 

```
make update-docs && git diff --quiet
```

Reply:
done